doc/jekyll_tools/fonts/rlogo.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
I guess we don't need the R logo to test the Python homepage :)

Reply:
R u sure? haha... removed!